sync/fromDisk/questions.js

```diff
@@ -24,6 +26,22 @@ module.exports.sync = function(courseInfo, questionDB, jobLogger, callback) {
         const questionsParam = Object.keys(questionDB).map(qid => {
             const q = questionDB[qid];
 
+            var thumbnailPath = null;
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.jpg'))) {
+                thumbnailPath = 'thumbnail.jpg';
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.png'))) {
+                thumbnailPath = 'thumbnail.png';
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.jpeg'))) {
+                thumbnailPath = 'thumbnail.jpeg';
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.svg'))) {
+                thumbnailPath = 'thumbnail.svg';
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.gif'))) {
+                thumbnailPath = 'thumbnail.gif';
+            }
```

Comment:
Please don't use `sync` functions anywhere in the code. High performance in NodeJS requires only using asynchronous functions like `fs.exists()` (not `fs.existsSync()`).

Reply:
Oh good catch.  We had this in there just to test the functionality with the intention of moving to async calls.